R/names.R

```diff
@@ -213,7 +213,7 @@ vec_names <- function(x,
 
   switch(arg_match(repair),
     minimal = minimal_names(x),
-    unique = as_unique_names(minimal_names(x), quiet = quiet),
+    unique = unique_names(x, quiet = quiet),
```

Comment:
Shouldn't some corresponding R code now disappear?

Reply:
This code is used for `repair = "universal"`, which remains implemented in R.